src/libraries/Microsoft.Bcl.Cryptography/src/System/Security/Cryptography/X509Certificates/X509CertificateKeyAccessors.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace System.Security.Cryptography.X509Certificates
+{
+    /// <summary>
+    /// Helper methods to access keys on <see cref="X509Certificate2"/>.
+    /// </summary>
+    public static class X509CertificateKeyAccessors
+    {
+        /// <summary>
+        ///   Gets the <see cref="SlhDsa"/> public key from this certificate.
+        /// </summary>
+        /// <param name="certificate">
+        ///   The X509 certificate that contains the public key.
+        /// </param>
+        /// <returns>
+        ///   The public key, or <see langword="null"/> if this certificate does not have an SLH-DSA public key.
+        /// </returns>
+        /// <exception cref="ArgumentNullException">
+        ///   <paramref name="certificate"/> is <see langword="null"/>.
+        /// </exception>
+        /// <exception cref="PlatformNotSupportedException">
+        ///   The certificate has an SLH-DSA public key, but the platform does not support SLH-DSA.
+        /// </exception>
+        /// <exception cref="CryptographicException">
+        ///   The public key was invalid, or otherwise could not be imported.
+        /// </exception>
+        [ExperimentalAttribute("SYSLIB5006")]
```

Comment:
This should use the const.

```C#
[Experimental(Experimentals.PostQuantumCryptographyDiagId)]
```

Reply:
If we put it on the class, but then mark ML-DSA (or whatever) as "not Experimental" while one of the rest still is, we have to be diligent at moving the Experimentals down to each remaining member.

We could _also_ put it on the class until there's one stable member so that we don't end up with "well, we deleted everything, because all of PQC got abandoned by the industry before the specs all went final-stable.... so now we have this empty static class"...  but I don't really see that outcome, so I don't feel like the class-level one is warranted.

Reply:
> We could also put it on the class until there's one stable member so that we don't end up with "well, we deleted everything, because all of PQC got abandoned by the industry before the specs all went final-stable.

More like, "It turns out that extensions are not the right shape for [whatever reason]" but, okay.